cmd/system.go

```diff
@@ -283,6 +283,18 @@ func CheckDependencies(logger *Logger) (bool, error) {
 				} else {
 					logger.Green("Library '%s' installed.", library.Name)
 				}
+			case RedHat:
+
+				installed, err := YumInstalled(library.Name)
+				if err != nil {
+					return false, err
+				}
+				if !installed {
+					errors = true
+					logger.Red("Library '%s' not found. %s", library.Name, library.Help)
```

Comment:
Could you please use logger.Error for this? Using Red was an error on the main branch. This has been fixed in v0.13.0-pre.

Reply:
sure ill change it now